### ISSUE TITLE
Do you use a single checkpoint to evaluate arbitrary-shot episodes?

### ISSUE THREAD

**Issue by dahyun-kang** [Fei-Long121/DeepBDC#3]
Hello,

Congratulations on your paper accepted as an oral presentation!

I would like to ask a simple question on the implementation:
To evaluate both 1-shot and 5-shot episodes on the test set, do you use one model checkpoint which had the best validation accuracy on 1-shot episodes?

Thank you for releasing the code publicly available!
Have a great day! 😃 

Best, 
Dahyun

**Comment by Fei-Long121** [comment]
Hi, thank you for your interests in our work. For 5-shot episodes, we also have used a  episodic meta-training.

**Comment by dahyun-kang** [comment]
Ah I see. It seems that you meta-train a K-shot model for each K-shot evaluation on the test set.
Thank you for the prompt reply.
Closing the issue.
